internal/cmd/database/create.go

```diff
@@ -45,6 +45,24 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			org, err := client.Organizations.Get(cmd.Context(), &ps.GetOrganizationRequest{
+				Organization: ch.Config.Organization,
+			})
+
+			if err != nil {
+				return err
+			}
+
+			if org.RemainingFreeDatabases == 0 {
+				ch.Printer.Printf("Organization [%s] does not have any free databases remaining\n", org.Name)
+				ch.Printer.Printf("If you choose to continue, this  database will be created on Scaler plan. The monthly cost is %v.\n", printer.BoldYellow("$29"))
```

Comment:
I see an extra space between "this" and "database" here.

Separately, having a "the" before "Scaler" may read a little better?